### PR TITLE
Lay out for active route link (connects #32)

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

File: src/app/home/navigation.component.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Router } from '@angular/router';
     selector:'main-navigation',
     template: `
         <ul>
-            <li *ngFor="let comp of components"><a [routerLink]="'/' + comp.link">{{comp.name.toUpperCase()}}</a></li>
+            <li *ngFor="let comp of components"><a routerLinkActive="active-link" [routerLink]="'/' + comp.link">{{comp.name.toUpperCase()}}</a></li>
```
Comment (nirojdyola) [inline src/app/home/navigation.component.ts]: Please remove space after </li> tag

File: src/app/home/navigation.scss [hunk 2]
```diff
@@ -35,5 +35,14 @@
     li:last-child a {
         border:none;
     }
-    
+
```
Comment (nirojdyola) [inline src/app/home/navigation.scss]: Extra line

File: src/app/home/navigation.component.ts (hunk 1 above)
Comment (razu9861) [reply src/app/home/navigation.component.ts]: extra spaced is been removed

File: src/app/home/navigation.scss (hunk 2 above)
Comment (razu9861) [reply src/app/home/navigation.scss]: Done

**Comment by razu9861** [comment]
screen shot 2017-09-08 at 3 22 56 pm

**Pull request closed without merging**